### PR TITLE
doc: Add documentation+FAQs for Parquet DataSource

### PR DESCRIPTION
**Pull request by Meghajit**
PR for https://github.com/odpf/dagger/issues/109

### CODE REVIEW

**Comment by Meghajit** [comment]
Added the configurations + some common gotchas for setting up configurations. Also, some FAQs and how to create the stream config for parquet. I haven’t touched the diagrams yet, as they will require some effort to be redrawn since the original resources are missing. However, have changed the description in relevent places.

File: docs/docs/guides/create_dagger.md [hunk 1]
```diff
@@ -6,13 +6,79 @@ This page contains how-to guides for creating a Dagger job and configure it.
...
+        - dt=2022-02-05
+            - hr=09
+                * g6agdasgd6asdgvadhsaasd829ajs.parquet
+                * . . . (more parquet files)
+            - (...more hour folders)
+        - (... more date folders)
+
+```
+
+or data partitioned, such as:
```
Comment (kevinbheda) [inline docs/docs/guides/create_dagger.md:38]: small typo here, I believe we went date partitioned here.

File: docs/docs/concepts/architecture.md [hunk 2]
```diff
@@ -10,9 +10,12 @@ Dagger or Data Aggregator is a cloud native framework for processing real-time s
 
 _**Stream**_
 
-- Stream are logical Data sources from which Dagger reads unbounded Data.
-- Dagger only supports Kafka Queue as the supported data source for now. In the context of Kafka, a single stream is a group of Kafka topics with the same schema. Dagger can support multiple streams at a time.
-- Dagger consumes the data points from Kafka. So many Kafka consumer-level configurations like consumer groups and auto offset reset can be set in the stream itself.
+- A Stream defines a logical grouping of a data source and its associated [`protobuf`](https://developers.google.com/protocol-buffers)
+  schema. All data produced by a source follows the protobuf schema. The source can be a bounded one such as `KAFKA_SOURCE` or `KAFKA_CONSUMER`
+  in which case, a single stream can consume from one or more topics all sharing the same schema. Otherwise, the source
+  can be an unbounded one such as `PARQUET_SOURCE` in which case, one or more parquet files as provided are consumed in a single stream.
```
[the remark this answers is not in the document]
Comment (kevinbheda) [reply docs/docs/concepts/architecture.md:16]: good catch

[the remark this answers is not in the document]
Comment (Meghajit) [reply docs/docs/concepts/architecture.md]: Fixed via commit https://github.com/odpf/dagger/pull/153/commits/7a2eec1c8cb38158d3f9737cc21bb7a5a22aab59

File: docs/docs/concepts/basics.md [hunk 3]
```diff
@@ -6,11 +6,19 @@ For Stream processing and hence for dagger user must know about some basic conce
...
+time as it is being produced. The source that is producing this data can either be a bounded source such as Parquet Files 
+or an unbounded source such as Kafka.
 
 ### Streams
 
-A group of Kafka topics sharing the same schema define a stream. The schema is defined using [`protobuf`](https://developers.google.com/protocol-buffers). You can have any number of schemas you want but the streaming queries become more complex with the addition of new schemas.
+A Stream defines a logical grouping of a data source and its associated [`protobuf`](https://developers.google.com/protocol-buffers) 
+schema. All data produced by a source follows the protobuf schema. The source can be a bounded one such as `KAFKA_SOURCE` or `KAFKA_CONSUMER` 
+in which case, a single stream can consume from one or more topics all sharing the same schema. Otherwise, the source 
+can be an unbounded one such as `PARQUET_SOURCE` in which case, one or more parquet files as provided are consumed in a single stream.
```
[the remark this answers is not in the document]
Comment (Meghajit) [reply docs/docs/concepts/basics.md]: Fixed via commit https://github.com/odpf/dagger/pull/153/commits/7a2eec1c8cb38158d3f9737cc21bb7a5a22aab59

File: docs/docs/guides/create_dagger.md (hunk 1 above)
Comment (Meghajit) [reply docs/docs/guides/create_dagger.md]: Fixed via commit https://github.com/odpf/dagger/pull/153/commits/31f5ec3aacaf808fc27ad70764169faddb77287d

**Pull request merged**